src/services/group.service.ts

```diff
@@ -31,7 +32,9 @@ export class GroupService {
       return newGroup.toDomain();
     }
 
-    throw `Group with login=${group.name} already exists`;
+    throw new IncorrectRequestToDBError(
```

Comment:
It is not possible to add a group with the same name. I think, to throw an error at that case is ok. Otherwise, what should do the addGroup method if the group with that name already exists?